multiples_3_5/multiples_3_5.rb

```diff
@@ -0,0 +1,4 @@
+def solution(number)
+  list = *(1...number).select { |n| (n % 3).zero? || (n % 5).zero? }
+  list.inject(0, :+)
+end
```

Comment:
`formatting nitpick`: 

`github` likes to have an empty line at the end of the file, this allows not to show a `changed line` in the `diff` if something is added at the end of the document.   

in `vscode` editor it can be automated with (`"files.insertFinalNewline": true`) config.

Reply:
I like to use .editorconfig file with settings per project. Most of editors are supported it.